packages/teleport-github-gateway/src/github-instance.ts

```diff
@@ -171,7 +219,7 @@ export default class GithubInstance {
         : await repo.createBlob(file.content)
 
     return {
-      sha: data.sha,
+      sha: file.status === 'deleted' ? null : data.sha,
```

Comment:
@ionutpasca we don't have `status` field in `file` object. You can either extend it here, if it's only needed in GitHub pacakage. Since, we don't use them in generators.

Reply:
Or you can check with `file.content` something ? 
```js
!file.content 
```
something of that sort if it works

Reply:
Ok, sounds good then. Can we update the types for GeneratedFolder and FeneratedFile to add optional status field in teleport-types. To pass this one

Reply:
My bad, didn't see that the build failed. Fixed now